test/tests.py

```diff
@@ -0,0 +1,143 @@
+from operations import create_index, check_index, create_document, delete_document, get_document, delete_index
+from operations import check_document
+from http import HTTPStatus
+import unittest
```

Comment:
In the readme, it sounds like you're running the tests with `pytest` and you're installing `pytest` in the requirements file. Is there a reason you're using `unittest` instead of `pytest` here?

Either one works, but if you're just using `unittest`, we might as well remove the external dependency on `pytest`.

Reply:
I preferred using `unittest` to implement the tests as these tests do not require any of the advantages that `pytest `provides over `unittest` and since its' way of running `setup and teardown` before _each and every_ test without having to explicitly mention that made the code look simpler and cleaner in my opinion.
BUT, the output of running the tests using `pytest` provides a lot more details in the way that unittest could not do, even with some extra configuration.